### PR TITLE
Terrarium: new recipe

### PR DESCRIPTION
**Pull request by nysnatuss**
Added [Terrarium](https://github.com/hydren/terrarium)'s recipe, a free and open source Terraria clone game. The recipe is based on the [building instructions](https://github.com/hydren/terrarium/blob/master/BUILDING.md) from the repo and uses SDL2.

Please correct any mistakes in the recipe, thanks!

### CODE REVIEW

**Comment by waddlesplash** [comment]
More than a little surprised it works with GCC2... but I guess you did not test this everywhere?

File: games-action/terrarium/terrarium-0.1.8.recipe [hunk 1]
```diff
@@ -0,0 +1,57 @@
+SUMMARY="Open source Terraria clone"
+DESCRIPTION="Terrarium is a free and open source Terraria clone game, possibly with some new features."
```
Comment (diversys) [inline games-action/terrarium/terrarium-0.1.8.recipe]: Wrap at 80th character.

File: games-action/terrarium/terrarium-0.1.8.recipe [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+{
+	cp -r build/linux-gcc-sdl2.0-release/ .
+	cd linux-gcc-sdl2.0-release
+	runConfigure ./configure.sh
+	make $jobArgs
+}
+
+INSTALL()
+{
+	mkdir -p $appsDir
```
Comment (diversys) [inline games-action/terrarium/terrarium-0.1.8.recipe]: mkdir -p $appsDir/Terrarium

File: games-action/terrarium/terrarium-0.1.8.recipe [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+	cp terrarium $appsDir/Terrarium
```
Comment (diversys) [inline games-action/terrarium/terrarium-0.1.8.recipe]: cp terrarium $appsDir/Terrarium/Terrarium

File: games-action/terrarium/terrarium-0.1.8.recipe [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+	cp -r resources $appsDir/Terrarium/resources/
+
+	addAppDeskbarSymlink $appsDir/terrarium Terrarium
```
Comment (diversys) [inline games-action/terrarium/terrarium-0.1.8.recipe]: addAppDeskbarSymlink $appsDir/Terrarium/Terrarium

**Comment by diversys** [comment]
The recipe is broken on both arches:
http://vmpkg.haiku-os.org/master/x86_64/logviewer.html?buildruns/691/builds/9162.log
http://vmpkg.haiku-os.org/master/x86_gcc2/logviewer.html?buildruns/543/builds/26671.log

**Pull request merged**